pkg/term/tc_other.go

```diff
@@ -1,21 +1,21 @@
 // +build !windows
-// +build !linux !cgo
-// +build !linux !ppc64le
```

Comment:
I think your "because" is  throwing me. I think we still need this to catch "other" (FreeBSD & OpenBSD?).